plugins/bulk_upload/csv/batch/BulkUploadCategoryUserEngineCsv.php

```diff
@@ -257,29 +255,38 @@ protected function createObjects()
 	 */
 	protected function createCategoryUserFromResultAndJobData (KalturaBulkUploadResultCategoryUser $bulkUploadCategoryUserResult)
 	{
-	    $categoryUser = new KalturaCategoryUser();
-	    //calculate parentId of the category
-	    
-	    if ($bulkUploadCategoryUserResult->categoryId)
-	    {
-	        $categoryUser->categoryId = $bulkUploadCategoryUserResult->categoryId;
-	    }
-	    else if ($this->categoryReferenceIdMap[$bulkUploadCategoryUserResult->categoryReferenceId])
-	    {
-	        $categoryUser->categoryId = $this->categoryReferenceIdMap[$bulkUploadCategoryUserResult->categoryReferenceId];
-	    }
-	    
-	    if ($bulkUploadCategoryUserResult->userId)
-	        $categoryUser->userId = $bulkUploadCategoryUserResult->userId;
-	        
-	    if (!is_null($bulkUploadCategoryUserResult->permissionLevel))
-	        $categoryUser->permissionLevel = $bulkUploadCategoryUserResult->permissionLevel;
-	        
-	    $categoryUser->updateMethod = KalturaUpdateMethodType::AUTOMATIC;
-	    if (!is_null($bulkUploadCategoryUserResult->updateMethod))
-	        $categoryUser->updateMethod = $bulkUploadCategoryUserResult->updateMethod; 
+		$categoryUser = new KalturaCategoryUser();
+		//calculate parentId of the category
+
+		if ($bulkUploadCategoryUserResult->categoryId)
+		{
+			$categoryUser->categoryId = $bulkUploadCategoryUserResult->categoryId;
+		}
+		else if ($this->categoryReferenceIdMap[$bulkUploadCategoryUserResult->categoryReferenceId])
+		{
+			$categoryUser->categoryId = $this->categoryReferenceIdMap[$bulkUploadCategoryUserResult->categoryReferenceId];
+		}
+
+		if ($bulkUploadCategoryUserResult->userId)
+		{
+			$categoryUser->userId = $bulkUploadCategoryUserResult->userId;
+		}
 	        
-	    return $categoryUser;
+		if (!is_null($bulkUploadCategoryUserResult->permissionLevel))
+		{
+			$categoryUser->permissionLevel = $bulkUploadCategoryUserResult->permissionLevel;
+		}
+		
+		if (!is_null($bulkUploadCategoryUserResult->updateMethod))
+		{
+			$categoryUser->updateMethod = $bulkUploadCategoryUserResult->updateMethod;
+		}
+		else
+		{
+			$categoryUser->updateMethod = KalturaUpdateMethodType::MANUAL;
```

Comment:
why this logic was changed?
until now the default was automatic in case no value was set up and not manual